ExpantaNum.js

```diff
@@ -520,7 +549,8 @@
     if (e<50000) e+=-139/51840*Math.pow(e,3);
     if (e<1e7) e+=1/288*Math.pow(e,2);
     if (e<1e20) e+=1/12*e;
-    return x.div(ExpantaNum.E).pow(x).mul(x.mul(ExpantaNum.PI).mul(2).sqrt()).times(errorFixer);
+    if (x.lte('1e157'))return x.div(ExpantaNum.E).pow(x).mul(x.mul(ExpantaNum.PI).mul(2).sqrt()).times(errorFixer);
+    else return x.gamma();
```

Comment:
Seems to work, though I would put the check before all the calculations. I wonder how you came up with `1e157`.